### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by oliverviljamaa**
Please comment on anything that's not clear.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```javascript
+import path from 'path';
+import TranslationHelper from '@transferwise/translation-helper'; // 1
+
+
+const messagesPath = path.join(__dirname, 'translations'); // 2
+
+const translationHelper = new TranslationHelper({
+  messagesPath, // required
+  messagesFileNameBase: 'my-messages', // default: 'messages'
```
Comment (coolsebz) [inline README.md]: in the documentation, you refer to this property as the `base file name`, curious why this isn't named `messagesBaseFileName` then 🤔

File: README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+This is a translation helper for Webpack.
+It is used to create specific bundles for each of your supported languages
+in addition to the bundle containing all the languages, automatically.
+This allows us to serve only the messages we need for a certain locale to the customers,
+decreasing the payload size.
+
+The setup is complicated, but after that,
+even with new language additions, no modifications are needed.
+
+## Use
```
Comment (coolsebz) [inline README.md]: maybe change this to `Usage`

File: README.md (hunk 1 above)
Comment (oliverviljamaa) [reply README.md]: Good point, there's a discrepancy.
Which do you think is more correct?
1. Base file name (indicates that file name includes file extension)
2. File name base (indicates that file name does not include file extension)

Comment (coolsebz) [reply README.md]: Fair enough, imho 2 makes more sense now that I have the context

Maybe it's a non issue and i'm just too picky but do you think having a 10th step in your description that shows an example input and output tree structure would help clear this up?

Comment (oliverviljamaa) [reply README.md]: The usage is complicated so nothing is too picky, will figure something out.

Comment (oliverviljamaa) [reply README.md]: Changed it to `messagesFileNameBase`☺️

**Pull request merged**